frog/imports/ui/GraphEditor/index.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import React, { Component } from 'react';
+import { Provider } from 'mobx-react';
+
+import { store } from './store';
+import EditorContainer from './EditorContainer';
+
+export default class AppClass extends Component {
+  componentWillMount() {
+    // store.ui.setId(assignGraph());
```

Comment:
Why did you comment this line? It causes the problem for me that we can edit a non existing graph on opening the Editor, which I think was one of your complaint on one of my previous PR :)
![screenshot from 2017-03-01 16-51-38](https://cloud.githubusercontent.com/assets/8818081/23469426/d7d5ad92-fea3-11e6-86d7-8e7f8c705a67.png)